ts/screens/wallet/payment/PaymentOutcomeCodeMessage.tsx

```diff
@@ -132,6 +133,8 @@ const PaymentOutcomeCodeMessage: React.FC<Props> = (props: Props) => {
   const learnMoreLink = "https://io.italia.it/faq/#pagamenti";
   const onLearnMore = () => openWebUrl(learnMoreLink);
 
+  useAvoidHardwareBackButton();
```

Comment:
The Jira ticket says that the flow should be closed on back button press, but this implementation only disables the hardware back button.
I believe we should instead intercept the back button press as follows:

```suggestion
  useHardwareBackButton(() => {
    props.navigateToWalletHome(props.shouldGoBackToEntrypointOnSuccess);
      return true;
    });
```

What do you think?

Reply:
Totally agree!

Addressed into 84182b75a133049a6b2985e7fc5247e87461544a